var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -39,24 +39,33 @@ class Legion(CMakePackage):
 
     variant('mpi', default=True,
             description='Build on top of mpi conduit for mpi inoperability')
+    variant('ibv', default=False,
+            description='Build on top of ibv conduit for InfiniBand support')
     variant('shared', default=True, description='Build shared libraries')
     variant('hdf5', default=True, description='Enable HDF5 support')
 
-    depends_on("cmake@3.1:", type='build')
+    depends_on("cmake@3.12.4", type='build')
```

Comment:
Huh a fixed version?

Reply:
well with the `:` it always uses the latest cmake, which may not necessarily build all the time

Reply:
Why not making it a range then?

Reply:
what do you mean a range? how do we do that?

Reply:
Something like `cmake@3.12.4:3.15`.

Reply:
So does it use 3.12.4 or 3.15?

Reply:
If you have experienced a problem with the latest CMake, this would make sense to constrain. If not, then IMO it's not worth adding an upper bound.

If you say 

`depends_on("cmake@3.12.4:3.15", type="build")`

It means that it can use any CMake version between 3.12.4 and 3.15 (inclusive).